src/animated-circle.ios.ts

```diff
@@ -1,25 +1,51 @@
 import { Common } from './animated-circle.common';
-import { Color } from 'tns-core-modules/color/Color';
+import { Color } from 'tns-core-modules/color';
+import { Label } from 'tns-core-modules/ui/label';
+import * as utils from 'tns-core-modules/utils/utils';
 
-declare const DRCircularProgressView;
 
 export class AnimatedCircle extends Common {
 
     private _ios: any;
 
+    private _label: UILabel;
+    private _text: string;
+
     constructor() {
         super();
         this._ios = DRCircularProgressView.new();
         this._ios.alternativeColor = new Color('#FF5722').ios;
         this._ios.progressColor = new Color('#3D8FF4').ios;
+        this._ios.backgroundColor = new Color('#348F74').ios;
         this._ios.thickness = 5;
+
+
+        this._label = UILabel.alloc().initWithFrame(CGRectMake(0, 0, 0, 0));
+        this._label.textAlignment = NSTextAlignment.Center;
+        this._label.textColor = new Color('#3D8FF4').ios;
```

Comment:
Can we make the text color configurable for the end user?

Reply:
it is configurable, check [animated-circle.ios.ts#L120](https://github.com/danielgek/nativescript-animated-circle/blob/feat-text/src/animated-circle.ios.ts#L120)